packages/plugin-lighthouse/src/lib/lighthouse-plugin.ts

```diff
@@ -1,25 +1,35 @@
-import { AuditOutputs, PluginConfig } from '@code-pushup/models';
+import { Audit, AuditOutputs, Group, PluginConfig } from '@code-pushup/models';
+import { filterByAuditSlug, filterBySlug } from '@code-pushup/utils';
 import { AUDITS, GROUPS, LIGHTHOUSE_PLUGIN_SLUG } from './constants';
+import { validateOnlyAudits } from './utils';
 
+type AuditSlug = (typeof AUDITS)[number]['slug'];
 export type LighthousePluginOptions = {
   url: string;
   outputPath?: string;
-  onlyAudits?: string | string[];
+  onlyAudits?: AuditSlug | AuditSlug[];
```

Comment:
As commented before, the `AuditSlug` is an over-complicated way of inferring a `string`. Just use `string` in the public API, you'll confuse the user less.

![image](https://github.com/code-pushup/cli/assets/34691111/8281c55c-2773-4274-af2f-9c305b3f2685)

```suggestion
  onlyAudits?: string | string[];
```